### DISCUSSION TITLE
Welcome to CogAlg Discussions!

### DISCUSSION THREAD

**Discussion by boris-kz** [boris-kz/CogAlg#39] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]


**Reply by lasred** [reply]
Hi Maksim

Thank you for your interest. It may be helpful to take a look at Boris's most recent comments/suggestions on this post - #55 where his recommendation was to start with line_patterns

This is analysis another engineer did of line_patterns - https://github.com/boris-kz/CogAlg/discussions/63

**Comment by boris-kz** [comment]
Thanks Maksim! Our current work is on line_PPs_draft in 1D alg and comp_slice_, comp_blob_draft in 2D alg. But to get there you need to understand lower modules first. I guess you already run frame_blobs, so you can continue to intra_blob, etc.? It's a lot more complex than 1D though, we normally start with line_patterns. But we currently more people getting into 1D alg, so you can try 2D for better balance.

**Comment by marat200073** [comment]
Hi.
I'm very delighted to join here.
I commented on my understanding and opinion about the CogAlgorithm in Discussion #57.
I want to keep thinking as well as contribute to developing CogAlgorithm with my knowledge and creativity.
Thanks.
Best Regards.
